src/infinite-scroll.js

```diff
@@ -131,7 +129,7 @@ angular.module(MODULE_NAME, [])
         return throttled;
       }
 
-      const handler = (THROTTLE_MILLISECONDS != null) ?
```

Comment:
Actually it needs to be let as it is resigned. es6lint doesn't allow for declaration before use.